docs/user_guide/emummc/sd_preparation.md

```diff
@@ -35,7 +35,7 @@ Atmosphere has its own bootloader, called fusee (primary). For the purposes of t
     2. Copy *the contents of* the Atmosphere `.zip` file to the root of your SD card
     3. Copy the `bootloader` folder from the Hekate `.zip` file to the root of your SD card
     4. Copy the `bootloader` folder from the `bootlogos.zip` file to the root of your SD card
-       - If you're asked to merge the bootloader folders, do so
+          - If you're asked to merge the bootloader folders and replace files, do so
```

Comment:
`*and* replace files`? You should probably make that `or replace the files`. This should be done for the sysmmc guide as well in the pr